apps/dsp-app/src/styles.scss

```diff
@@ -1,4 +1,282 @@
-/* You can add global styles to this file, and also import other style files */
+// Include non-theme styles for core.
+@use '@angular/material' as mat;
+@use "styles/config" as *;
+@use "styles/config";
+@use "styles/font";
+@use "styles/theme";
+@use "styles/layout";
+@use "styles/elements";
+@use "styles/search";
+@use "styles/viewer";
+@use "styles/ontology";
 
-@use "material-theme";
-@use "main";
+/* TODO(mdc-migration): Remove legacy-core once all legacy components are migrated*/
+@include mat.legacy-core();
+@include mat.core();
+
+$custom-button-typography-level: mat.define-typography-level(
+        $font-size: 16px,
+        $line-height: 24px,
+        $font-weight: 600,
+        $font-family: 'Roboto, "Helvetica Neue", sans-serif',
+        $letter-spacing: normal
+);
+$custom-typography: mat.define-typography-config(
+        $button: $custom-button-typography-level
+);
+
+// Include all typography configurations for the components (now separate from mat.core())
+/* TODO(mdc-migration): Remove all-legacy-component-typographies once all legacy components are migrated*/
+@include mat.all-legacy-component-typographies();
+@include mat.all-component-typographies($custom-typography);
+
+// Define the color palettes for the theme.
+$primary: mat.define-palette(config.$blue_palette, 700, 500, 900);
+$accent: mat.define-palette(config.$light_blue_palette, 900, 600, 900);
+
+// The warn palette is optional (defaults to red).
+$warn: mat.define-palette(config.$velvet_red_palette, 700, 500, 900);
+
+// Define a theme.
+$theme: mat.define-light-theme((
+        color: (primary: $primary, accent: $accent, warn: $warn),
+        density: 0
+));
+
+// Include all theme styles for the components.
+/* TODO(mdc-migration): Remove all-legacy-component-themes once all legacy components are migrated*/
+@include mat.all-legacy-component-themes($theme);
+@include mat.all-component-themes($theme);
+
+
+body {
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
+  margin: 0;
+}
+
+// material icons
+.material-icons {
+  font-family: 'Material Icons';
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px; /* Preferred icon size */
+  display: inline-block;
+  line-height: 1;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: normal;
+  white-space: nowrap;
+  direction: ltr;
+
+  /* Support for all WebKit browsers. */
+  -webkit-font-smoothing: antialiased;
+  /* Support for Safari and Chrome. */
+  text-rendering: optimizeLegibility;
+
+  /* Support for Firefox. */
+  -moz-osx-font-smoothing: grayscale;
+
+  /* Support for IE. */
+  font-feature-settings: 'liga';
+}
+
+// The following styles will override material design!
+
+// jdnconvertiblecalendar main style
+.mat-datepicker-content {
+  .mat-calendar {
+    height: auto !important;
+  }
+}
+
+/* TODO(mdc-migration): The following rule targets internal classes of card that may no longer apply for the MDC version.*/
```

Comment:
You should check if this is doing something still. If it is, you can delete the TODO comment, otherwise delete the css.

Reply:
This has to do with grid. Is grid already migrated?